ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -2537,3 +2616,130 @@ export const testScrollInsideBox = (): Promise<boolean> =>
       reject(error);
     }
   });
+
+export const testJustifyContent = (): Promise<boolean> =>
```

Comment:
What is the difference between this test and testAllignItems?tThey both seems to have `alignItems` and `justifyContent` configuration. Do we need both tests?